[Bug] [Module Name] vertica sink 使用generate_sink_sql无法通过主键进行更新

### Search before asking

- [x] I had searched in the [issues](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.


### What happened

sink config中：
generate_sink_sql = true
primary_keys = ["id"]
vertica jdbc使用8.x、12.x、24.x，都会在实际插入数据时，不管主键是否存在重复值，都会报错：
Caused by: com.vertica.util.ClientSyntaxErrorException: [Vertica][VJDBC](2848) ERROR: Could not determine data type of parameter $1
上述报错是在将表中字段全部改成string之后的，若是表中存在int类型或date类型字段，报错稍有不同：
Caused by: java.sql.SQLException: [Vertica][VJDBC](https://github.com/apache/seatunnel/3376) ERROR: Failed to find conversion function from unknown to date
或
Caused by: com.vertica.support.exceptions.ErrorException: [Vertica][VJDBC](https://github.com/apache/seatunnel/3376) ERROR: Failed to find conversion function from unknown to int


### SeaTunnel Version

2.3.9

### SeaTunnel Config

```conf
env {
    parallelism = 1
    checkpoint.interval = 120000
    job.name= "kafka-seatunnel-vertia"
    job.mode= "STREAMING"
}

source {
    kafka {
        plugin_output = "kafka_out"
        bootstrap.servers = "127.0.0.1:9092,127.0.0.2:9092,127.0.0.3:9092"
        topic = "input-test"
        consumer.group = "test-group"
        format = json
        partition-discovery.interval-millis = 60000
        commit_on_checkpoint = true
        start_mode = group_offsets
        kafka.config = {
            client.id = test-group
            max.poll.records = 10
            auto.offset.reset = "latest"
            enable.auto.commit = "false"
        }
        schema = {
            fields {
                id = "string"
                gtoken = "string"
                gname = "string"
                device_score = "string"
                device_level = "string"
                detecttime = "string"
            }
        }
    }
}


sink {
    jdbc {
        plugin_input = "kafka_out"
        url = "jdbc:vertica://127.0.0.1:5433/dev"
        driver = "com.vertica.jdbc.Driver"
        user = "xxxxxxxx"
        password = "xxxxxxxxx"
        generate_sink_sql = true
        primary_keys = ["id"]
        database = "tmp"
        table = "test_table"
        batch_size = 1
        batch_interval_ms = 30000
    }
}
```

### Running Command

```shell
./bin/seatunnel.sh --config ./config/kafka.template -m local
```

### Error Exception

```log
org.apache.seatunnel.connectors.seatunnel.jdbc.exception.JdbcConnectorException: ErrorCode:[COMMON-08], ErrorDescription:[Sql operation failed, such as (execute,addBatch,close) etc...] - unable to open JDBC writer
        at org.apache.seatunnel.connectors.seatunnel.jdbc.internal.JdbcOutputFormat.createAndOpenStatementExecutor(JdbcOutputFormat.java:107) ~[connector-jdbc-2.3.9-2.12.15-em.jar:2.3.9]
        at org.apache.seatunnel.connectors.seatunnel.jdbc.internal.JdbcOutputFormat.open(JdbcOutputFormat.java:79) ~[connector-jdbc-2.3.9-2.12.15-em.jar:2.3.9]
        at org.apache.seatunnel.connectors.seatunnel.jdbc.sink.JdbcSinkWriter.tryOpen(JdbcSinkWriter.java:122) ~[connector-jdbc-2.3.9-2.12.15-em.jar:2.3.9]
        at org.apache.seatunnel.connectors.seatunnel.jdbc.sink.JdbcSinkWriter.write(JdbcSinkWriter.java:133) ~[connector-jdbc-2.3.9-2.12.15-em.jar:2.3.9]
        at org.apache.seatunnel.connectors.seatunnel.jdbc.sink.JdbcSinkWriter.write(JdbcSinkWriter.java:43) ~[connector-jdbc-2.3.9-2.12.15-em.jar:2.3.9]
        at org.apache.seatunnel.api.sink.multitablesink.MultiTableWriterRunnable.run(MultiTableWriterRunnable.java:67) [seatunnel-starter.jar:2.3.9]
        at org.apache.seatunnel.api.tracing.MDCRunnable.run(MDCRunnable.java:43) [seatunnel-starter.jar:2.3.9]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_111]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_111]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [?:1.8.0_111]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [?:1.8.0_111]
        at java.lang.Thread.run(Thread.java:745) [?:1.8.0_111]
Caused by: java.sql.SQLSyntaxErrorException: [Vertica][VJDBC](2848) ERROR: Could not determine data type of parameter $1
        at com.vertica.util.ServerErrorData.buildException(Unknown Source) ~[vertica-jdbc-24.2.0-1.jar:?]
        at com.vertica.io.ProtocolStream.readExpectedMessage(Unknown Source) ~[vertica-jdbc-24.2.0-1.jar:?]
        at com.vertica.dataengine.VDataEngine.prepareImpl(Unknown Source) ~[vertica-jdbc-24.2.0-1.jar:?]
        at com.vertica.dataengine.VDataEngine.prepare(Unknown Source) ~[vertica-jdbc-24.2.0-1.jar:?]
        at com.vertica.dataengine.VDataEngine.prepare(Unknown Source) ~[vertica-jdbc-24.2.0-1.jar:?]
        at com.vertica.jdbc.common.SPreparedStatement.<init>(SPreparedStatement.java:310) ~[vertica-jdbc-24.2.0-1.jar:?]
        at com.vertica.jdbc.jdbc41.S41PreparedStatement.<init>(S41PreparedStatement.java:91) ~[vertica-jdbc-24.2.0-1.jar:?]
        at com.vertica.jdbc.jdbc42.S42PreparedStatement.<init>(S42PreparedStatement.java:77) ~[vertica-jdbc-24.2.0-1.jar:?]
        at com.vertica.jdbc.VerticaJdbc42PreparedStatementImpl.<init>(Unknown Source) ~[vertica-jdbc-24.2.0-1.jar:?]
        at com.vertica.jdbc.VJDBCObjectFactory.createPreparedStatement(Unknown Source) ~[vertica-jdbc-24.2.0-1.jar:?]
        at com.vertica.jdbc.common.SConnection.prepareStatement(SConnection.java:1309) ~[vertica-jdbc-24.2.0-1.jar:?]
        at com.vertica.jdbc.common.SConnection.prepareStatement(SConnection.java:1263) ~[vertica-jdbc-24.2.0-1.jar:?]
        at org.apache.seatunnel.shade.com.zaxxer.hikari.pool.ProxyConnection.prepareStatement(ProxyConnection.java:337) ~[connector-jdbc-2.3.9-2.12.15-em.jar:2.3.9]
        at org.apache.seatunnel.shade.com.zaxxer.hikari.pool.HikariProxyConnection.prepareStatement(HikariProxyConnection.java) ~[connector-jdbc-2.3.9-2.12.15-em.jar:2.3.9]
        at org.apache.seatunnel.connectors.seatunnel.jdbc.internal.executor.FieldNamedPreparedStatement.prepareStatement(FieldNamedPreparedStatement.java:673) ~[connector-jdbc-2.3.9-2.12.15-em.jar:2.3.9]
        at org.apache.seatunnel.connectors.seatunnel.jdbc.internal.JdbcOutputFormatBuilder.lambda$createSimpleExecutor$6(JdbcOutputFormatBuilder.java:348) ~[connector-jdbc-2.3.9-2.12.15-em.jar:2.3.9]
        at org.apache.seatunnel.connectors.seatunnel.jdbc.internal.executor.SimpleBatchStatementExecutor.prepareStatements(SimpleBatchStatementExecutor.java:43) ~[connector-jdbc-2.3.9-2.12.15-em.jar:2.3.9]
        at org.apache.seatunnel.connectors.seatunnel.jdbc.internal.executor.BufferReducedBatchStatementExecutor.prepareStatements(BufferReducedBatchStatementExecutor.java:50) ~[connector-jdbc-2.3.9-2.12.15-em.jar:2.3.9]
        at org.apache.seatunnel.connectors.seatunnel.jdbc.internal.JdbcOutputFormat.createAndOpenStatementExecutor(JdbcOutputFormat.java:105) ~[connector-jdbc-2.3.9-2.12.15-em.jar:2.3.9]
        ... 11 more
```

### Zeta or Flink or Spark Version

zeta

### Java or Scala Version

_No response_

### Screenshots

_No response_

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/JdbcOutputFormatBuilder.java
+++ b/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/JdbcOutputFormatBuilder.java
@@ -187,8 +187,7 @@ public class JdbcOutputFormatBuilder {
         }
         if (enableUpsert) {
             Optional<String> upsertSQL =
-                    dialect.getUpsertStatement(
-                            database, table, tableSchema.getFieldNames(), pkNames);
+                    dialect.getUpsertStatementByTableSchema(database, table, tableSchema, pkNames);
             if (upsertSQL.isPresent()) {
                 return createSimpleExecutor(
                         upsertSQL.get(),

--- a/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/JdbcDialect.java
+++ b/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/JdbcDialect.java
@@ -18,6 +18,7 @@
 package org.apache.seatunnel.connectors.seatunnel.jdbc.internal.dialect;
 
 import org.apache.seatunnel.api.table.catalog.TablePath;
+import org.apache.seatunnel.api.table.catalog.TableSchema;
 import org.apache.seatunnel.api.table.converter.BasicTypeDefine;
 import org.apache.seatunnel.api.table.converter.TypeConverter;
 import org.apache.seatunnel.api.table.schema.event.AlterTableAddColumnEvent;
@@ -238,6 +239,25 @@ public interface JdbcDialect extends Serializable {
      */
     Optional<String> getUpsertStatement(
             String database, String tableName, String[] fieldNames, String[] uniqueKeyFields);
+
+    /**
+     * Constructs the dialects upsert statement if supported; such as MySQL's {@code DUPLICATE KEY
+     * UPDATE}, or PostgreSQL's {@code ON CONFLICT... DO UPDATE SET..}.
+     *
+     * <p>If supported, the returned string will be used as a {@link java.sql.PreparedStatement}.
+     * Fields in the statement must be in the same order as the {@code columns in tableSchema}
+     * parameter.
+     *
+     * <p>If the dialect does not support native upsert statements, the writer will fallback to
+     * {@code SELECT ROW Exists} + {@code UPDATE}/{@code INSERT} which may have poor performance.
+     *
+     * @return the dialects {@code UPSERT} statement or {@link Optional#empty()}.
+     */
+    default Optional<String> getUpsertStatementByTableSchema(
+            String database, String tableName, TableSchema tableSchema, String[] uniqueKeyFields) {
+        return getUpsertStatement(
+                database, tableName, tableSchema.getFieldNames(), uniqueKeyFields);
+    }
 
     /**
      * Different dialects optimize their PreparedStatement

--- a/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/vertica/VerticaDialect.java
+++ b/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/vertica/VerticaDialect.java
@@ -65,6 +65,7 @@ public class VerticaDialect implements JdbcDialect {
                 Arrays.stream(fieldNames)
                         .filter(fieldName -> !Arrays.asList(uniqueKeyFields).contains(fieldName))
                         .collect(Collectors.toList());
+        // vertica jdbc currently needs to display the specified type to execute
         String valuesBinding =
                 tableSchema.getColumns().stream()
                         .map(

--- a/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/vertica/VerticaDialect.java
+++ b/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/vertica/VerticaDialect.java
@@ -65,7 +65,7 @@ public class VerticaDialect implements JdbcDialect {
                 Arrays.stream(fieldNames)
                         .filter(fieldName -> !Arrays.asList(uniqueKeyFields).contains(fieldName))
                         .collect(Collectors.toList());
-        // vertica jdbc currently needs to display the specified type to execute
+        // Vertica JDBC currently requires explicitly specifying the data type
         String valuesBinding =
                 tableSchema.getColumns().stream()
                         .map(

--- a/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/vertica/VerticaDialect.java
+++ b/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/vertica/VerticaDialect.java
@@ -17,6 +17,7 @@
 
 package org.apache.seatunnel.connectors.seatunnel.jdbc.internal.dialect.vertica;
 
+import org.apache.seatunnel.api.table.catalog.TableSchema;
 import org.apache.seatunnel.connectors.seatunnel.jdbc.internal.converter.JdbcRowConverter;
 import org.apache.seatunnel.connectors.seatunnel.jdbc.internal.dialect.DatabaseIdentifier;
 import org.apache.seatunnel.connectors.seatunnel.jdbc.internal.dialect.JdbcDialect;
@@ -53,16 +54,35 @@ public class VerticaDialect implements JdbcDialect {
     @Override
     public Optional<String> getUpsertStatement(
             String database, String tableName, String[] fieldNames, String[] uniqueKeyFields) {
+        return Optional.empty();
+    }
+
+    @Override
+    public Optional<String> getUpsertStatementByTableSchema(
+            String database, String tableName, TableSchema tableSchema, String[] uniqueKeyFields) {
+        String[] fieldNames = tableSchema.getFieldNames();
         List<String> nonUniqueKeyFields =
                 Arrays.stream(fieldNames)
                         .filter(fieldName -> !Arrays.asList(uniqueKeyFields).contains(fieldName))
                         .collect(Collectors.toList());
         String valuesBinding =
-                Arrays.stream(fieldNames)
-                        .map(fieldName -> ":" + fieldName + " " + quoteIdentifier(fieldName))
+                tableSchema.getColumns().stream()
+                        .map(
+                                column -> {
+                                    String fieldName = column.getName();
+                                    String sourceType = column.getSourceType();
+                                    return "CAST("
+                                            + ":"
+                                            + fieldName
+                                            + " AS "
+                                            + sourceType
+                                            + ")"
+                                            + " AS "
+                                            + quoteIdentifier(fieldName);
+                                })
                         .collect(Collectors.joining(", "));
 
-        String usingClause = String.format("SELECT %s FROM DUAL", valuesBinding);
+        String usingClause = String.format("SELECT %s ", valuesBinding);
         String onConditions =
                 Arrays.stream(uniqueKeyFields)
                         .map(
@@ -77,7 +97,7 @@ public class VerticaDialect implements JdbcDialect {
                         .map(
                                 fieldName ->
                                         String.format(
-                                                "TARGET.%s=SOURCE.%s",
+                                                "%s=SOURCE.%s",
                                                 quoteIdentifier(fieldName),
                                                 quoteIdentifier(fieldName)))
                         .collect(Collectors.joining(", "));

--- a/seatunnel-connectors-v2/connector-jdbc/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/vertica/VerticaDialectTest.java
+++ b/seatunnel-connectors-v2/connector-jdbc/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/vertica/VerticaDialectTest.java
@@ -1,0 +1,124 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.connectors.seatunnel.jdbc.internal.dialect.vertica;
+
+import org.apache.seatunnel.shade.com.google.common.collect.Lists;
+
+import org.apache.seatunnel.api.table.catalog.ConstraintKey;
+import org.apache.seatunnel.api.table.catalog.PhysicalColumn;
+import org.apache.seatunnel.api.table.catalog.PrimaryKey;
+import org.apache.seatunnel.api.table.catalog.TableSchema;
+import org.apache.seatunnel.api.table.type.BasicType;
+import org.apache.seatunnel.api.table.type.LocalTimeType;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+import java.util.HashMap;
+
+public class VerticaDialectTest {
+
+    @Test
+    void testUpsertStatementByTableSchema() {
+        final String dataBaseName = "test_database";
+        final String tableName = "test_table";
+        TableSchema tableSchema =
+                TableSchema.builder()
+                        .column(
+                                PhysicalColumn.of(
+                                        "id",
+                                        BasicType.LONG_TYPE,
+                                        22L,
+                                        0,
+                                        false,
+                                        null,
+                                        "id",
+                                        "BIGINT",
+                                        new HashMap<>()))
+                        .column(
+                                PhysicalColumn.of(
+                                        "name",
+                                        BasicType.STRING_TYPE,
+                                        128L,
+                                        0,
+                                        false,
+                                        null,
+                                        "name",
+                                        "VARCHAR",
+                                        new HashMap<>()))
+                        .column(
+                                PhysicalColumn.of(
+                                        "age",
+                                        BasicType.INT_TYPE,
+                                        (Long) null,
+                                        0,
+                                        true,
+                                        null,
+                                        "age",
+                                        "INT",
+                                        new HashMap<>()))
+                        .column(
+                                PhysicalColumn.of(
+                                        "createTime",
+                                        LocalTimeType.LOCAL_DATE_TIME_TYPE,
+                                        3L,
+                                        0,
+                                        true,
+                                        null,
+                                        "createTime",
+                                        "TIME",
+                                        new HashMap<>()))
+                        .primaryKey(PrimaryKey.of("id", Lists.newArrayList("id")))
+                        .constraintKey(
+                                Collections.singletonList(
+                                        ConstraintKey.of(
+                                                ConstraintKey.ConstraintType.INDEX_KEY,
+                                                "name",
+                                                Lists.newArrayList(
+                                                        ConstraintKey.ConstraintKeyColumn.of(
+                                                                "name", null)))))
+                        .build();
+
+        VerticaDialect dialect = new VerticaDialect();
+        final String[] doUpdateKeyFields = {"id"};
+        final String[] doNothingKeyFields = {"id", "name", "age"};
+
+        String doUpdateSql =
+                dialect.getUpsertStatementByTableSchema(
+                                dataBaseName, tableName, tableSchema, doUpdateKeyFields)
+                        .orElseThrow(
+                                () ->
+                                        new AssertionError(
+                                                "Expected doUpdateSql String to be present"));
+        Assertions.assertEquals(
+                doUpdateSql,
+                " MERGE INTO test_database.\"test_table\" TARGET USING (SELECT CAST(:id AS BIGINT) AS \"id\", CAST(:name AS VARCHAR) AS \"name\", CAST(:age AS INT) AS \"age\", CAST(:createTime AS TIME) AS \"createTime\" ) SOURCE ON (TARGET.\"id\"=SOURCE.\"id\")  WHEN MATCHED THEN UPDATE SET \"name\"=SOURCE.\"name\", \"age\"=SOURCE.\"age\", \"createTime\"=SOURCE.\"createTime\" WHEN NOT MATCHED THEN INSERT (\"id\", \"name\", \"age\", \"createTime\") VALUES (SOURCE.\"id\", SOURCE.\"name\", SOURCE.\"age\", SOURCE.\"createTime\")");
+
+        String doNothingSql =
+                dialect.getUpsertStatementByTableSchema(
+                                dataBaseName, tableName, tableSchema, doNothingKeyFields)
+                        .orElseThrow(
+                                () ->
+                                        new AssertionError(
+                                                "Expected doNothingSql String to be present"));
+        Assertions.assertEquals(
+                doNothingSql,
+                " MERGE INTO test_database.\"test_table\" TARGET USING (SELECT CAST(:id AS BIGINT) AS \"id\", CAST(:name AS VARCHAR) AS \"name\", CAST(:age AS INT) AS \"age\", CAST(:createTime AS TIME) AS \"createTime\" ) SOURCE ON (TARGET.\"id\"=SOURCE.\"id\" AND TARGET.\"name\"=SOURCE.\"name\" AND TARGET.\"age\"=SOURCE.\"age\")  WHEN MATCHED THEN UPDATE SET \"createTime\"=SOURCE.\"createTime\" WHEN NOT MATCHED THEN INSERT (\"id\", \"name\", \"age\", \"createTime\") VALUES (SOURCE.\"id\", SOURCE.\"name\", SOURCE.\"age\", SOURCE.\"createTime\")");
+    }
+}

--- a/seatunnel-connectors-v2/connector-jdbc/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/vertica/VerticaDialectTest.java
+++ b/seatunnel-connectors-v2/connector-jdbc/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/vertica/VerticaDialectTest.java
@@ -110,7 +110,7 @@ public class VerticaDialectTest {
                 doUpdateSql,
                 " MERGE INTO test_database.\"test_table\" TARGET USING (SELECT CAST(:id AS BIGINT) AS \"id\", CAST(:name AS VARCHAR) AS \"name\", CAST(:age AS INT) AS \"age\", CAST(:createTime AS TIME) AS \"createTime\" ) SOURCE ON (TARGET.\"id\"=SOURCE.\"id\")  WHEN MATCHED THEN UPDATE SET \"name\"=SOURCE.\"name\", \"age\"=SOURCE.\"age\", \"createTime\"=SOURCE.\"createTime\" WHEN NOT MATCHED THEN INSERT (\"id\", \"name\", \"age\", \"createTime\") VALUES (SOURCE.\"id\", SOURCE.\"name\", SOURCE.\"age\", SOURCE.\"createTime\")");
 
-        String doNothingSql =
+        String upsertCreateTimeSQL =
                 dialect.getUpsertStatementByTableSchema(
                                 dataBaseName, tableName, tableSchema, doNothingKeyFields)
                         .orElseThrow(
@@ -118,7 +118,7 @@ public class VerticaDialectTest {
                                         new AssertionError(
                                                 "Expected doNothingSql String to be present"));
         Assertions.assertEquals(
-                doNothingSql,
+                upsertCreateTimeSQL,
                 " MERGE INTO test_database.\"test_table\" TARGET USING (SELECT CAST(:id AS BIGINT) AS \"id\", CAST(:name AS VARCHAR) AS \"name\", CAST(:age AS INT) AS \"age\", CAST(:createTime AS TIME) AS \"createTime\" ) SOURCE ON (TARGET.\"id\"=SOURCE.\"id\" AND TARGET.\"name\"=SOURCE.\"name\" AND TARGET.\"age\"=SOURCE.\"age\")  WHEN MATCHED THEN UPDATE SET \"createTime\"=SOURCE.\"createTime\" WHEN NOT MATCHED THEN INSERT (\"id\", \"name\", \"age\", \"createTime\") VALUES (SOURCE.\"id\", SOURCE.\"name\", SOURCE.\"age\", SOURCE.\"createTime\")");
     }
 }

